molo/profiles/admin.py

```diff
@@ -5,12 +5,12 @@
 from django.contrib.auth.models import User
 from django.contrib.auth.admin import UserAdmin
 from django.contrib.admin.sites import NotRegistered
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
 from django.utils.timezone import localtime
 from django.conf import settings
 from django.db.models import Q
 
-from daterange_filter.filter import DateRangeFilter
+# from daterange_filter.filter import DateRangeFilter
```

Comment:
Why are we removing this filter?

Reply:
not compatibale, looking at alternatives